src/test/resources/yaml/user/topic.yml

```diff
@@ -0,0 +1,10 @@
+topic1:
+  id: 1
```

Comment:
Good that this thing came out.
It looks like that we _forced_ to define PK explicitly to be able to find something by PK in tests.
This fact neutralizes a few very powerful features of JFixtures - automatic PK generating and automatic PK->FK relationships resolving.
It seems that we need to extend JFixtures's API to be able to get the PK by the row alias:
```java
RowAliases aliases = JFixtures.sql99("/path").aliases();
Long id = aliases.forTable("topic").row("topic1");
```
WDYT?